src/core/tsi/alts/handshaker/alts_tsi_handshaker.h

```diff
@@ -45,6 +45,8 @@ typedef struct alts_tsi_handshaker alts_tsi_handshaker;
  *
  * - options: ALTS credentials options containing information passed from TSI
  *   caller (e.g., rpc protocol versions).
+ * - resource: an alts_shared_resource object that is shared by all ALTS TSI
```

Comment:
Please note that the caller is responsible for ensuring that the resources continue to exist until after the handshaker has been destroyed.

Reply:
Done.